app/forms/state_file/nj_disabled_exemption_form.rb

```diff
@@ -0,0 +1,14 @@
+module StateFile
+  class NjDisabledExemptionForm < QuestionsForm
+    set_attributes_for :intake,
+                       :primary_disabled,
+                       :spouse_disabled
+    
+    validates :primary_disabled, presence: true, if: -> { !intake.direct_file_data.is_primary_blind? }
+    validates :spouse_disabled, presence: true, if: -> { intake.filing_status_mfj? && !intake.direct_file_data.is_spouse_blind? }
+    def save
+      @intake.update(attributes_for(:intake))
+    end
```

Comment:
NIT: I think, but am not sure if, this is the default behavior. You may consider deleting it and trying to relevant test and seeing if it still works.

Reply:
I had misremembered this. I double checked and I was thinking of `::from_intake`. Sorry for the noise!

Reply:
Unfortunately, I think it is required. We found a bug the other day when leaving the `save` function undefined - see: https://github.com/codeforamerica/vita-min/pull/4854